scss/mixins/_forms.scss

```diff
@@ -1,55 +1,51 @@
-//DEPRECATE 1.6
```

Comment:
This mixin is now a combination of itself and the other `fd-form-base` mixin. All styles are intact (hopefully 😉 )